packages/data/src/types.ts

```diff
@@ -31,6 +31,7 @@ export interface FlowAccessors<F> {
   getFlowTime?: FlowAccessor<F, Date>; // TODO: use number instead of Date
   // getFlowColor?: FlowAccessor<string | undefined>;
   getFlowAggFunc: FlowAggregatorFunc<number[], number>;
+  getFlowAggWeight: FlowAccessor<F, number>;
```

Comment:
Here is the new wight function that similar to the magnitude function is fed into flowmap.